drivers/sensor/sun8i-thermal.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <debug.h>
+#include <error.h>
+#include <limits.h>
+#include <mmio.h>
+#include <sensor.h>
+#include <sensor/sun8i-thermal.h>
+
+#define SENSOR_COUNT                3
```

Comment:
Instead of a macro, add another entry to the end of the enum below. See `axp803` for an example

Reply:
done